pkgs/applications/video/clapper/default.nix

```diff
@@ -58,20 +54,19 @@ stdenv.mkDerivation rec {
     gtk4
     libGL
     libadwaita
-    libsoup
+    libsoup_3
     wayland
     wayland-protocols
+    vala
+    libmicrodns
+    desktop-file-utils # for update-desktop-database
+    shared-mime-info # for update-mime-database
   ];
 
   postPatch = ''
     patchShebangs build-aux/meson/postinstall.py
   '';
 
-  postInstall = ''
-    cp ${src}/data/icons/*.svg $out/share/icons/hicolor/scalable/apps/
-    cp ${src}/data/icons/*.svg $out/share/icons/hicolor/symbolic/apps/
-  '';
-
   meta = with lib; {
     description = "A GNOME media player built using GJS with GTK4 toolkit and powered by GStreamer with OpenGL rendering. ";
```

Comment:
```suggestion
    description = "A GNOME media player built using GJS with GTK4 toolkit and powered by GStreamer with OpenGL rendering";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/306548/commits/2fd6717b51137e2ab74b11cb4117332e3f69e4ad